### ISSUE TITLE
[?] Bullets Penetrate Vehicles

### ISSUE THREAD

**Issue by Doughnut2001** [InZidiuZ/op-framework-issue-tracker#1622]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes I Have.

**Summary:**
Getting bullets to penetrate vehicles just enough to be able to hit the passengers but not behind the car itslef

**Reason:**
So far, most of the shootouts are in one of two places, Rooftops and Cars
rooftops don't seem to be an issue at all since that can be resolved with actual skill, but car shootouts are a different story since PD doesn't have automatic weapons that can be shot from cars, 
meaning that an officer in the ground with an SMG will always lose to a suspect on an EVO with a full auto skorpion because of:
-Better firepower
-Moving cover
-Faster Speed

and even in vehicle combat it's mostly a full auto gun vs a combat pistol

I've been on plenty of shootouts where they have a car with full auto weapons going around people in rooftops and being able to shoot down officers that are on foot because of the latter.

There's also some vehicles that have very small windows that don't let you shoot through them, making people inside invincible, ergo backing up and shooting, or shooting at rooftops from inside of a vehicle using the roof for cover

Summing up, vehicle combat is broken for PD, 4 officers can't do anything against one guy with an uzi if we have no cover.
and realistically, cars just don't even stop bullets.

In my opinion: removing the ability to take cover inside of a car would drastically shift the vehicle combat meta in a way that would force people to think about better ways to engage in gunplay, therefore balancing the Police vs Criminal combat, this is a healthy way to fix the vehicle combat without fully removing it.

LINK FOR REFERENCE:
https://www.gta5-mods.com/misc/bullets-penetrate-cars

**Comment by InZidiuZ** [comment]
Closing as outdated. If this is still relevant, please create a new issue.
